File: baekjoon/16000/16693.cpp
#include <stdio.h>
#define PI 3.14159265358979323846

int main()
{
    double a, r, p1, p2;
    scanf("%lf %lf %lf %lf", &a, &p1, &r, &p2);
    double slice = a / p1, whole = PI * r * r / p2;
    if (slice > whole) printf("Slice of ");
    else printf("Whole ");
    printf("pizza");
    return 0;
}

File: baekjoon/16000/16693.py
import math


def main():
    a, p1 = map(int, input().split())
    r, p2 = map(int, input().split())
    slice, whole = a / p1, math.pi * r**2 / p2
    print(f"{'Slice of' if slice > whole else 'Whole'} pizza")


if __name__ == "__main__":
    main()
